### PR TITLE
Update travis for 0.5 and add coveralls & codecov

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ os:
   - osx
 julia:
   - 0.4
+  - 0.5
   - nightly
 notifications:
   email: false
+after_success:
+  - julia -e 'cd(Pkg.dir("BinDeps")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder()); Codecov.submit(Codecov.process_folder())';
```
Comment (musm) [inline .travis.yml]: Is this prefered over:

```julia
 - julia -e 'cd(Pkg.dir("BinDeps")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
 - julia -e 'cd(Pkg.dir("BinDeps")); Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())';
```

Comment (ararslan) [inline .travis.yml]: I think what you have now is fine.

**Comment by musm** [comment]
Is this ok?

**Comment by musm** [comment]
Companion pr https://github.com/JuliaLang/BinDeps.jl/pull/255

**Pull request merged**